nxt/cli.py

```diff
@@ -216,7 +216,9 @@ def main():
                                default='')
     test_parser = subs.add_parser('test', help='Runs unit tests')
     test_parser.set_defaults(which='test')
-
+    if '--' in sys.argv:
+        idx = sys.argv.index('--')
+        sys.argv = sys.argv[idx:]
```

Comment:
Could be missing something in my understanding, but this sounds like it would break input like
`nxt --verbose --gui` or `nxt -v --convert --replace` since it would trim what args are found based on `--`.

Reply:
The `--` flag ends up being an empty arg. In Blender's case (and I would assume other applications), it ignores all flags after an empty `--` flag.
[How I found out about this](https://blender.stackexchange.com/a/8405)
[Blender CLI docs about --](https://docs.blender.org/manual/en/dev/advanced/command_line/arguments.html#misc-options)